facade/service.go

```diff
@@ -2056,6 +2056,18 @@ func (f *Facade) WaitService(ctx datastore.Context, dstate service.DesiredState,
 				return result.Err
 			}
 		case <-timeoutC:
+			if dstate == service.SVCPause {
```

Comment:
How about we change the error message to "Error waiting for services" and go ahead and do it on any timeout (not just Pause)